unity-renderer/Assets/DCLServices/WearablesCatalogService/LambdasWearablesCatalogService.cs

```diff
@@ -53,6 +53,54 @@ public void Dispose()
             Clear();
         }
 
+        public async UniTask<(IReadOnlyList<WearableItem> wearables, int totalAmount)> RequestOwnedWearablesAsync(
+            string userId, int pageNumber, int pageSize, CancellationToken cancellationToken, string category = null,
+            NftRarity rarity = NftRarity.None, ICollection<string> collectionIds = null,
+            string name = null, (NftOrderByOperation type, bool directionAscendent)? orderBy = null)
+        {
+            var queryParams = new List<(string name, string value)>
```

Comment:
We would be allocating memory en every call. Maybe we can reuse the same list and a mechanism to know which params to take (maybe a bitmask) or ignore the ones with a "null" value in the key.

Reply:
I could cache the list and do a clearing before adding the new params. Im not wary about allocations because the frequency of calls is somewhat low, and the size of the list contains 8 elements at most. I think that the bitmask/param filtering gain is not worth due the complexity addition to the code